hydra-node/src/Hydra/Chain/Offline.hs

```diff
@@ -36,6 +39,15 @@ offlineHeadId = UnsafeHeadId "offline"
 offlineHeadSeed :: HeadSeed
 offlineHeadSeed = UnsafeHeadSeed "offline"
 
+data InitialUTxOParseException = InitialUTxOParseException String
+  deriving stock (Show)
+
+instance Exception InitialUTxOParseException where
+  displayException (InitialUTxOParseException err)
+    = "Failed to parse initial UTXO: " <> err <> ". Example UTXO: "
```

Comment:
can we not hardcode the error example?

Reply:
maybe its ok to keep it for the time being.

Reply:
What would be a good alternative?

Reply:
Oh, do you mean, linking the documentation on the UTxO format somewhere else? I'd be in support of that, but I'm currently not sure where to link or how else to explain it